Tests/MapboxCoreNavigationTests/PassiveLocationDataSourceTests.swift

```diff
@@ -69,13 +69,9 @@ class PassiveLocationDataSourceTests: XCTestCase {
         let filePathURL: URL = URL(fileURLWithPath: bundle.bundlePath.appending("/tiles/liechtenstein"))
 
         // Create PassiveLocationDataSource and configure it with the tiles version (version is used to find the tiles in the cache folder)
+        Navigator.tilesVersion = tilesVersion
```

Comment:
Is this change really going to affect underlying navigator configuration? I think if shared navigator was created elsewhere underlying properties will change, but Navigator will still use previous ones.

Reply:
Yes, this test will have side effects in Navigator that would affect any other test in this class. We can move the settings to `setUpWithError()` to ensure consistency regardless of execution order, but otherwise I don’t think we can be any more robust, because ultimately Navigator’s underlying `MapboxNavigationNative.Navigator` is also a singleton that lasts for the remainder of the process.